index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>HTML Basics</title>
+        <style>
+            body {
+                position: center;
+                color: white;
+                font-family: Arial, Helvetica, sans-serif;
+
+                }
+
+            body::before {
+                    content: "";
+                    position: fixed;
+                    top: 0;
+                    left: 0;
+                    width: 100%;
+                    height: 100%;
+                    background-image: url("https://images.wallpapersden.com/image/download/minimalist-black-digital-blend_a2pnam2UmZqaraWkpJRobWllrWdma2U.jpg");
+                    background-position: center;
+                    background-size: cover;
+                    background-repeat: no-repeat;
+                    background-attachment: fixed;
+                    filter: blur(5px);
+                    z-index: -1;
+                }
+
+            body {
+                    text-align: center left;
+                }
+
+            .round-image {
+                    position: center;
+                    border-radius: 50%;
+                    width: 200px;
+                    height: 200px;
+                    border: 3px solid #CC5500;
+                }
+            .round-image2 {
+                    border-radius: 30px;
+                    position: center;
+                    border: 3px solid #CC5500
+                }    
+
+            h1 {text-align: center;} 
+            p {text-align: center;}
+            div {text-align: center;}
+            
+               
+            b, strong {
+                color: #CC5500;
+                }
+
+            div.container {
+                text-align: center;
+                }
+
+            ul {
+                display: inline-block;
+                text-align: left;
+                }
+
+            ol {display: inline-block;
+                text-align: left;
+                }
+            div.italic
+
+            {color: #2491cf;
+
+            }            
+        </style>    
+    </head>
+    <body>
+    <div class="content">    
+    <h1>📘 The Code Magazine</h1> 
+    <h2>The Basic Language of the Web: HTML</h2>
+        <img src="img/laura-jones.jpg" alt="Cute Groot" title="Me Groot" class="round-image" />
+        <p>Posted by <b>Laura Jones</b> on Monday, June 21st 2027</p>
+        <br>
+        <img src="img/post-img.jpg" alt="Coding" title="Coding" class="round-image2" />
+        <br>
+        <br>
+        <p>All modern websites and web applications are built using three <i>fundamental</i> technologies: HTML, CSS and JavaScript. These are the languages of the web.</p>
+        <p>In this post, let's focus on HTML. We will learn what HTML is all about, and why you too should learn it.</p>
+        <br>
+    <h3><b>What is HTML?</b></h3>
```

Comment:
No need to use ```<b>``` tag inside an ```<h>``` since all headings are bold by default. But that's not an issue, just an advice to keep in mind for future